### ISSUE TITLE
openslideload integration

### ISSUE THREAD

**Issue by ajay1685** [kleisauke/net-vips#18]
I am getting an error for 
NetVips.Image.Openslideload(fileName) 

Exception thrown: 'NetVips.VipsException' in NetVips.dll
An unhandled exception of type 'NetVips.VipsException' occurred in NetVips.dll
no such operation openslideload
VipsOperation: class "openslideload" not found

**Comment by kleisauke** [comment]
It looks like your libvips has been compiled without OpenSlide support. If you need OpenSlide support in NetVips, you need to install libvips yourself and include OpenSlide support. This is easy to do on linux or macOS hosts.

On Windows you'll need to use a globally-installed libvips. Set the `UseGlobalLibvips` property to `true`:
```xml
<PropertyGroup>
  <UseGlobalLibvips>true</UseGlobalLibvips>
</PropertyGroup>
```
And download the `vips-dev-w64-all-x.y.z.zip` libvips binary. You'll need to add the `vips-dev-x.y.z\bin` directory to your `PATH` environment variable. I've written a complete tutorial [here](https://github.com/kleisauke/net-vips/issues/3#issuecomment-386236773).

**Comment by ajay1685** [comment]
> 
> 
> It looks like your libvips has been compiled without OpenSlide support. If you need OpenSlide support in NetVips, you need to install libvips yourself and include OpenSlide support. This is easy to do on linux or macOS hosts.
> 
> On Windows you'll need to use a globally-installed libvips. Set the `UseGlobalLibvips` property to `true`:
> 
> ```
> <PropertyGroup>
>   <UseGlobalLibvips>true</UseGlobalLibvips>
> </PropertyGroup>
> ```
> 
> And download the `vips-dev-w64-all-x.y.z.zip` libvips binary. You'll need to add the `vips-dev-x.y.z\bin` directory to your `PATH` environment variable. I've written a complete tutorial [here](https://github.com/kleisauke/net-vips/issues/3#issuecomment-386236773).

I am using a pre compiled binary available at the lib-vips project. I am not sure it has openslide support built in !

**Comment by kleisauke** [comment]
Make sure you download the libvips **`-all`** distribution instead than the `-web`. The `-all` distribution contains ALL the file format loaders that libvips supports, including OpenSlide, and a complete copy of ImageMagick. It can process many different file types, but it is also rather vulnerable to hacking.

You could also try to download the latest pre-compiled libvips Windows binaries from [this repo](https://github.com/libvips/build-win64-mxe/releases). The advantage of this distribution is that all dependencies are updated to the latest version (where possible), so it'll reduce the attack surface.

**Comment by ajay1685** [comment]
Thank you for helping. I will give your recommendations a try. I am a begginer at programming using various libraries. So I don't know how it all comes together , I have a few years of experience manipulating and visualizing data in Matlab.

**Comment by ajay1685** [comment]
Here is my what I am doing
var im = NetVips.Image.NewFromFile("J:\Admin\Desktop\123.svs");
im.Tiffsave("J:\\Admin\\Desktop\\testx2.tif", "lzw");

NetVips.VipsException: 'unable to call tiffsave
J:\Admin\Desktop\123.svs: Compression scheme 33005 tile decoding is not implemented

**Comment by ajay1685** [comment]
Excellent !! Thank you for testing it out for me. I will try to troubleshoot the issue on my end. I running Visual studio 2017.

And yes , I did try VIPS.exe at command line with affine option and it worked just as it should.

And yes, the svs files pack a lot of data , I have to deal with them on daily basis , the largest one I decoded so far was 30GB. 

Libvips with net-vips opens up lot of possibilities for me to handle those files, this just the beggining for me , I have a long way to go, and lot to learn along the way.

Thank you again.
